src/service/standard-actions-impl.js

```diff
@@ -129,6 +129,16 @@ export class StandardActions {
           user().error(TAG, e.message);
         });
 
+      case 'scrollTo':
+        user().assert(args['id'],
+            'AMP target scrollTo must provide element ID');
+        const element = getAmpdoc(node).getElementById(args['id']);
+        user().assert(element,
+            'AMP target scrollTo must provide element ID present on page');
+        invocation.node = element;
+        this.handleScrollTo(invocation);
+        return null;
```

Comment:
Please return the result of `this.handleScrollTo(invocation)`. That methods returns a promise which allows for chaining, I think this one should do the same.